sdk/download.go

```diff
@@ -50,15 +52,38 @@ func TarballURL(version string) string {
 	return u.String()
 }
 
-func DownloadFile(file, url string) error {
-	plog.Infof("Downloading %s to %s", url, file)
+func DownloadFile(file, fileURL string, client *http.Client) error {
+	plog.Infof("Downloading %s to %s", fileURL, file)
 
+	// handle bucket urls by using api to get media link
+	parseURL, err := url.Parse(fileURL)
+	if err != nil {
+		return err
+	}
+	if parseURL.Scheme == "gs" {
+		if client == nil {
+			return fmt.Errorf("DownloadFile requires non-nil client with gs:// links")
```

Comment:
Nevermind, I can't delete the logic because you need client to get the medialink. Still, just using default client seems right here.